api/src/graphql/outputTemplates.sdl.ts

```diff
@@ -4,22 +4,42 @@ export const schema = gql`
     name: String!
     version: String!
     effectiveDate: DateTime!
-    rulesGeneratedAt: DateTime!
+    rulesGeneratedAt: DateTime
     createdAt: DateTime!
     updatedAt: DateTime!
     reportingPeriods: [ReportingPeriod]!
   }
 
+  type SignedUrls {
+    CPF1A: String
+    CPF1B: String
+    CPF1C: String
+    CPFSubrecipient: String
+  }
+
+  type OutputTemplateWithSignedUrl {
+    outputTemplate: OutputTemplate!
+    signedUrls: SignedUrls!
+  }
+
   type Query {
     outputTemplates: [OutputTemplate!]! @requireAuth
     outputTemplate(id: Int!): OutputTemplate @requireAuth
   }
 
+  input FileNames {
+    CPF1A: String
```

Comment:
Wondering if we should make each of these non-null, since it seems like we expect them to be populated in `outputTemplates.ts` below?